composer.json

```diff
@@ -25,7 +25,6 @@
     "suggest": {
         "symfony/yaml": "Required if you'd like to serialize data to YAML format."
     },
-    "minimum-stability": "dev",
```

Comment:
Oh yes, I was thinking this line was added and not removed.

I have to stop PR review before my morning coffee. :coffee: :wink: